### PR TITLE
fix(crons): Always assigne seat with ACCEPTED_FOR_UPSERT

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Previous to this change we would only try to assign a seat when a
monitor was upserted.  We should actually ALWAYS try to assign a seat
even if the monitor already exists.

This is an unusual case since there should already be a billing seat for
a monitor if it exists. This can happen when billing seats become out of
sync with he monitor in bug scenarios. This change hardens against that.

Fixes https://github.com/getsentry/team-crons/issues/170

### CODE REVIEW

**Comment by mbauer404** [comment]
@evanpurkhiser Thanks for getting this up! We need to hold on merging it until the backfills are completed.

**Comment by evanpurkhiser** [comment]
Sounds good. Thanks @mbauer404

File: tests/sentry/monitors/test_monitor_consumer.py [hunk 1]
```diff
@@ -733,3 +733,31 @@ def test_monitor_accept_upsert_no_seat(
...
+
+    @mock.patch("sentry.quotas.backend.assign_monitor_seat")
+    @mock.patch("sentry.quotas.backend.check_accept_monitor_checkin")
+    def test_monitor_accept_upsert_existing_monitor(
+        self,
+        check_accept_monitor_checkin,
+        assign_monitor_seat,
+    ):
+        """
+        Validate the unusual casse where a seat does not already exist but a
```
Comment (wedamija) [inline tests/sentry/monitors/test_monitor_consumer.py:745]: Nit: case

**Review by wedamija — APPROVED** [review]

**Comment by mbauer404** [comment]
> @evanpurkhiser Thanks for getting this up! We need to hold on merging it until the backfills are completed.

Both the backfills have been run, we can merge this now

**Review by mbauer404 — APPROVED** [review]

**Pull request merged**